src/main/java/io/lettuce/core/resource/DefaultClientResources.java

```diff
@@ -70,9 +71,11 @@
  * <li>a {@code socketAddressResolver} which is a provided instance of {@link SocketAddressResolver}.</li>
  * <li>a {@code timer} that is a provided instance of {@link io.netty.util.HashedWheelTimer}.</li>
  * <li>a {@code tracing} that is a provided instance of {@link Tracing}.</li>
+ * <li>a {@code addressResolverGroup} that is a provided instance of {@link AddressResolverGroup}.</li>
```

Comment:
Nit: We try to sort elements alphabetically.

Reply:
(nits) also i noticed it should be `an {@code addressResolverGroup} ...`